deployment/helm/memory-qos/templates/_helpers.tpl

```diff
@@ -0,0 +1,17 @@
+{{/*
+Common labels
+*/}}
+{{- define "memory-qos.labels" -}}
+helm.sh/chart: {{ .Chart.Name }}-{{ .Chart.Version }}
+app.kubernetes.io/name: nri-memory-qos
+app.kubernetes.io/instance: {{ .Release.Name }}
```

Comment:
nit: I think we could include the selectorLabels here to keep these always automatically in sync. Like:

```suggestion
{{ include "memory-qos.selectorLabels" . }}
```

Reply:
Good point. Fixed